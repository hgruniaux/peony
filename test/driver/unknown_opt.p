//# RUN: peony -unknown-opt
//~! ERROR: unknown command-line option '-unknown-opt'
