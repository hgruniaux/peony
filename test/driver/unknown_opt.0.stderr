error: unknown command-line option '-unknown-opt'
